crates/kitsune_p2p/mdns/examples/discover.rs

```diff
@@ -0,0 +1,24 @@
+use kitsune_mdns::*;
+use futures_util::{pin_mut, stream::StreamExt, self};
+
+#[async_std::main]
+async fn main() {
+   // Debug info
+   let mut builder = env_logger::Builder::new();
```

Comment:
Did this example get updated? It still has aysnc_std && env_logger.

Reply:
oops. My bad. Fixed.